tests/e2e/lib.sh

```diff
@@ -153,6 +153,8 @@ export_test_environment() {
     ci_export ROX_SCANNER_V4 "${ROX_SCANNER_V4:-false}"
     ci_export ROX_CLOUD_SOURCES "${ROX_CLOUD_SOURCES:-true}"
     ci_export ROX_AUTH_MACHINE_TO_MACHINE "${ROX_AUTH_MACHINE_TO_MACHINE:-true}"
+    ci_export ROX_REGISTRY_RESPONSE_TIMEOUT "${ROX_REGISTRY_RESPONSE_TIMEOUT:-90s}"
+    ci_export ROX_REGISTRY_CLIENT_TIMEOUT "${ROX_REGISTRY_RESPONSE_TIMEOUT:-120s}"
```

Comment:
Probably this part is the most important
I don't remember failures with go tests and we do not have metrics for them :( but we will have!
- #10242